go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -1039,3 +1039,22 @@ func (h *defaultNetworkControllerEventHandler) SyncFunc(objs []interface{}) erro
 func (h *defaultNetworkControllerEventHandler) IsObjectInTerminalState(obj interface{}) bool {
 	return h.baseHandler.isObjectInTerminalState(h.objType, obj)
 }
+
+// DeleteAlreadyProcessed returns true if the object has already been processed for deletion
+func (h *defaultNetworkControllerEventHandler) DeleteAlreadyProcessed(obj interface{}) bool {
+	switch h.objType {
+	case factory.PodType,
+		factory.PeerPodSelectorType,
+		factory.PeerPodForNamespaceAndPodSelectorType,
+		factory.LocalPodSelectorType,
+		factory.EgressIPPodType:
+		pod := obj.(*kapi.Pod)
+		key := util.GetLogicalPortName(pod.Namespace, pod.Name)
+		// If the pod still exists in the logical port cache we know we never attempted to delete it.
+		// This is because first thing that happens during a pod delete event is the port is removed from the cache
+		return h.oc.logicalPortCache.isPortRemoved(key)
```

Comment:
what is PodHandler failed and didn't add port to the lspCache, but other handlers succeeded. Then delete event comes, and all handlers ignore it?

Reply:
The other handlers should rely on lspCache, so if they dont see the port in there they shouldn't have done anything.